### PR TITLE
feat: Added Everskies, fix(Viu, Naver): Made it faster

### LINKED ISSUE #3649: EverSkies | everskies.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 

**What's the name of the service?** https://everskies.com/ 

**What should the Presence display?** on forums, on profile, on another persons profile, playing games, on the shops 

**If possible, please provide a logo for the service (512x512 minimum)** 
image

### PR DESCRIPTION
**Pull request by Slowlife01**
Resolves #3649 

## Everskies:
image
image
image
image
image

## Viu:
image

## Naver:
image
image

### CODE REVIEW

File: websites/E/Everskies/presence.ts [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      setPresenceData(){
+        const searchParams = new URLSearchParams(document.location.search);
+        presenceData.details = "Browsing users...";
+
+        if (document.location.search.startsWith("?")){
+          if (searchParams.get("friends"))
+            presenceData.details = "Viewing their friends";
+          else if (searchParams.get("q")){
+            presenceData.details = "Searching for user:";
+            presenceData.state = searchParams.get("q");
```
Comment (ririxidev) [inline websites/E/Everskies/presence.ts:92]:
```suggestion
          presenceData.details = searchParams.get("friends") ? "Viewing their friends" : "Searching for user:"
          if(searchParams.get("q") presenceData.state = searchParams.get("q");
```

File: websites/E/Everskies/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "thumbnail": "https://i.imgur.com/scedVmN.png",
+    "version": "1.0.0",
+    "color": "#c9bfff",
+    "tags": [
+      "everskies",
+      "forums"
+    ],
+    "url": "everskies.com",
+    "description": {
+      "en": "Everskies is a dress-up community with focus on creativity!"
```
Comment (ririxidev) [inline websites/E/Everskies/dist/metadata.json:19]:
```suggestion
      "en": "A community site for teens and young adults with focus on creativity!"
```
I literally think that's their official desc like on twitter 🤷

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/E/Everskies/dist/metadata.json (hunk 2 above)
Comment (SlowLife1661) [reply websites/E/Everskies/dist/metadata.json:19]: idk

**Review by ririxidev — APPROVED** [review]
yes i did broke i feel very bad :(

**Review by ririxidev — APPROVED** [review]
ok good now?

**Review by Bas950 — APPROVED** [review]

**Pull request merged**